src/pu-main.c

```diff
@@ -82,16 +92,7 @@ main(G_GNUC_UNUSED int argc,
         return 1;
     }
 
-    if (arg_debug) {
-        const gchar *domains = g_getenv("G_MESSAGES_DEBUG");
-
-        if (domains != NULL) {
-            g_autofree gchar *new_domains = g_strdup_printf("%s %s", domains, prog_name);
-            g_setenv("G_MESSAGES_DEBUG", new_domains, TRUE);
-        } else {
-            g_setenv("G_MESSAGES_DEBUG", prog_name, TRUE);
-        }
-    }
+    pu_log_setup(arg_quiet, arg_debug, arg_debug_domains);
```

Comment:
How do you enable debug messages for *all* modules explicitly? Using `-D all` doesn't seem to work. `G_MESSAGES_DEBUG=all` allows to enable all modules.

Reply:
Enable debug for all modules is still `-d`. I'll add that to the description.